readme.md

```diff
@@ -159,7 +159,7 @@ Jira notifications are messages sent to a Mattermost channel when a particular e
 
 Notifications and webhooks can be used together or you can opt for one of them.
 
-![This is a channel notification of a new bug that was created in Jira](https://github.com/mattermost/mattermost-plugin-jira/assets/74422101/e7020c3e-48f6-4825-8193-6a189f6c96eb)
+![This is a channel notification of a new bug that was created in Jira](https://github.com/mattermost/mattermost-plugin-jira/assets/19710961/fc19bb59-260f-4904-8887-b8de41421bf6)
```

Comment:
This screenshot is actually not showing a channel notification. It's instead showing the "Attach to Jira Issue" feature. So the original screenshot was incorrect for that reason as well. I think we can use the screenshot in the repo's README for this image.

Reply:
@mickmister I've updated the PR with your suggested changes.

Reply:
@chriswachira - I believe 2 screenshots are expected as part of this PR. One which you've updated and then another showing a channel notification.

Reply:
@cwarnermm I've updated the PNG file under assets showing the new location for the Jira actions and I've also updated the screenshot link in the readme section to display the `ticket-created.png` image under assets which shows a Jira channel notification on Mattermost

Reply:
Thanks, @chriswachira! It was a misunderstanding on my part! I expected to see two image assets, but was able to verify the second asset by URL. Nicely done on this PR, and thank you so much!